src/thorin/fe/lexer.cpp

```diff
@@ -275,6 +275,7 @@ void Lexer::parse_digits(int base /*= 10*/) {
     }
 }
 
+// TODO do we need this?
```

Comment:
https://github.com/search?q=repo%3AAnyDSL/thorin2%20_any&type=code
(Although the search can be unreliable.)

I don't think so. (One would need to test what depends on the function)